client/post-editor/editor-drawer/index.jsx

```diff
@@ -240,7 +242,23 @@ const EditorDrawer = React.createClass( {
 					isNew={ this.props.isNew }
 				/>
 			</AccordionSection>
-		)
+		);
+	},
+
+	renderSeo: function() {
+		if ( ! config.isEnabled( 'manage/advanced-seo' ) || ! this.props.site ) {
```

Comment:
>Another option would be to show this accordion for everyone, and show a promotional message if they don't have the business plan.

I'm nowhere close to being a marketing person, but that sounds like a reasonable upsell idea to me.

Otherwise (and perhaps for the near-term), I agree we shouldn't show the accordion and have it disappear after sites are loaded if the site is not upgraded.